cnf-certification-test/performance/suite.go

```diff
@@ -79,6 +85,22 @@ var _ = ginkgo.Describe(common.PerformanceTestKey, func() {
 	// Scheduling related tests ends here
 })
 
+func testLimitedUseOfExecProbes(env *provider.TestEnvironment) {
+	var compliantObjects []*testhelper.ReportObject
+	var nonCompliantObjects []*testhelper.ReportObject
+	for _, put := range env.Pods {
+		for _, cut := range put.Containers {
+			if cut.LivenessProbe.PeriodSeconds < 10 && cut.LivenessProbe.InitialDelaySeconds <= 10 {
```

Comment:
This is about limiting the number and frequency of exec probes. Readiness, liveliness and startup probes can all  be exec probes. Consider using existing function func (c *Container) HasExecProbes() bool to only count exec probes. 
Why test InitialDelaySeconds? 
Also, there is a missing test. There should be no more that 10 exec probes (of any type) in the cluster.